distribution/src/resources/config-tool/deployment-full.toml

```diff
@@ -47,7 +47,7 @@ properties.isCascadeDeleteEnabled = true    #inferred
 
 [user_store]
 #usr-mgt.xml
-type = "read_only_ldap" # inferred default read_only_ldap
+type = "read_only_ldap" # inferred default read_only_ldap # OR  read_write_ldap
 class = "org.wso2.micro.integrator.security.user.core.ldap.ReadOnlyLDAPUserStoreManager" # inferred
```

Comment:
let's remove this entry, may be in a separate commit. This is an advanced configuration.

Reply:
since this is full toml , does it matters ?